aries_cloudagent/protocols/issue_credential/v2_0/formats/vc_di/handler.py

```diff
@@ -253,7 +253,35 @@ async def _create():
         if not cred_offer:
             cred_offer = await _create()
 
-        return self.get_format_data(CRED_20_OFFER, cred_offer)
+        vcdi_cred_offer = {
```

Comment:
thank you @ianco !  for production do you think we should move this into the issuer, perhaps in a VcdiIssuer analogous to IndyIssuer, or an alternate create_credential_offer function?

Reply:
After some thought I think this should be in the vc_di handler class and you shouldn't add new methods to the AnonCredsIssuer (unless they are direcrtly related to a method you need to call in anoncreds-rs).  In this case I think the same cred offer is used and it's just a matter of formatting the attachment for the credential exchange.